packages/core/src/components/menu-category/menu-category.tsx

```diff
@@ -126,6 +126,10 @@ export class MenuCategory {
         this.showItems = true;
         this.showDropdown = false;
       },
+      complete: () => {
+        this.menuItemsContainer!.removeAttribute('max-height');
```

Comment:
Just out of curiosity:

Is this line necessary considering that the `max-height` will get set in the `setMenuItemsContainerMaxHeight` fn immediately afterwards?